example/lib/features/example/features/method/processors/example_google_pay_processor.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:collection/collection.dart';
+import 'package:example/core/constants.dart';
+import 'package:example/core/ui/snackbar.dart';
+import 'package:example/data/models/pay_method.dart';
+import 'package:example/data/models/pay_method_type.dart';
+import 'package:example/data/models/product.dart';
+import 'package:payu/payu.dart';
+
+class ExampleGooglePayProcessor {
+  final List<Product> _products;
+  final PayuMobilePayments _service;
+
+  ExampleGooglePayProcessor(this._products, this._service);
+
+  void process({required GooglePay googlePay, required Function(PayMethod) onProcessed}) async {
+    final totalPrice = _products.map((e) => (e.quantity * e.price)).sum / 100;
```

Comment:
może tworzyć problemy z zaokrągleniem, nie lepiej żeby to było intem?

Reply:
!!

Reply:
@apachucy To jest przykład, więc merchant może zrobić formatowanie na różne sposoby.